JustSaying.UnitTests/AwsTools/MessageHandling/MessageDispatcherTests/WhenDispatchingMessage.cs

```diff
@@ -167,4 +253,19 @@ public void ShouldLogException()
             }
         }
     }
+
+    public class NonRetryableException : Exception, INonRetryable
+    {
+        public NonRetryableException(string message) : base(message)
+        {
+        }
+
+        public NonRetryableException(string message, Exception innerException) : base(message, innerException)
+        {
+        }
+
+        public NonRetryableException()
```

Comment:
Should provide a default message for this exception by calling `: base("some message")`

Reply:
Happy to do this but it is only for the Test.